src/organisms/hintsAndHovers/styles.tsx

```diff
@@ -11,5 +11,6 @@ export const Tooltip = Styled(ReactTooltip)`
   max-width: 296px;
   &.__react_component_tooltip {
     border-radius: 8px;
+    opacity: 1 !important;
```

Comment:
@shan8851 why do we need !important ?

Reply:
We don't need this, was testing something out with `hintsAndHovers` and forgot to remove. Thanks for spotting it!